File: app/src/main/java/edu/ksu/phenoapps/verify/SettingsFragment.java
package edu.ksu.phenoapps.verify;

import android.os.Bundle;
import android.preference.PreferenceFragment;

public class SettingsFragment extends PreferenceFragment {

    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        addPreferencesFromResource(edu.ksu.phenoapps.verify.R.xml.preferences);
    }
}

File: app/src/main/java/org/phenoapps/intercross/SettingsFragment.kt
package org.phenoapps.intercross

import android.os.Bundle
import android.preference.PreferenceFragment

class SettingsFragment : PreferenceFragment() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        addPreferencesFromResource(org.phenoapps.intercross.R.xml.preferences)


    }
}
